responsibleai/responsibleai/feature_metadata.py

```diff
@@ -36,8 +36,6 @@ def __init__(self, identity_feature_name: Optional[str] = None,
             warnings.warn('datetime_features are not in use currently.')
         if self.categorical_features is not None:
```

Comment:
This kind of begs the question of why we accept them in the first place. Could be commented out completely, or only added whenever they are indeed used.
I guess that's not an issue with this PR at all, just the larger direction. Let's have this conversation next week when people are back.

Reply:
Sounds good!